check-synced.sh

```diff
@@ -69,9 +69,9 @@ fi
     #^The prometheus RPC's num-running-compactions key doesn't seem to correspond to actual
     # compaction events, so we'll determine compaction by another, dumber but accurate method:
     chk_numlines=100000 #Look through the last 100,000 lines of the db LOG
-    log_file="$e_db_path/LOG"
```

Comment:
Using a hard-coded path to the log file makes it impossible to parse the log file for the testnet4 instance which lives in a different path.